src/ProgramBuilder.ts

```diff
@@ -210,6 +213,9 @@ export class ProgramBuilder {
 
         //on any file watcher event
         this.watcher.on('all', async (event: string, thePath: string) => { //eslint-disable-line @typescript-eslint/no-misused-promises
+            if (!this.program) {
+                throw new Error('TILT: somehow file watcher ran before `ProgramBuilder.run()`');
```

Comment:
what does "TILT" mean?

Reply:
I don't remember where I picked that up-- it's the message that a pinball machine displays before it locks up when it detects a player is cheating. Maybe "Internal invariant violation error" would be better?

Reply:
I renamed this to hopefully make it more clear.

Reply:
Yep, much better!